lib/src/codec/udp/udp_message_decoder.dart

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
Perhaps we could shorten to `lib/src/codec/udp/message_decoder.dart`

Reply:
Similarly for other files under `.../udp`, it's implicit by the folder they are in that they are dealing with UDP.